config/table-headers.js

```diff
@@ -683,3 +691,40 @@ export const CONFIGURED_RECEIVER = {
   formatter:     'Link',
   formatterOpts: { options: { internal: true } },
 };
+
+export const ACCESS_KEY = {
+  name:     'name',
+  labelKey: 'tableHeaders.accessKey',
+  align:    'left',
+  sort:     ['name'],
+  width:    200,
+};
+
+export const SCOPE = {
+  name:        'scope',
+  value:       'clusterName',
+  labelKey:    'tableHeaders.scope',
+  dashIfEmpty: true,
+  align:       'left',
+  sort:        ['scope'],
+  width:       100,
+};
+
+export const EXPIRES_AT = {
+  name:          'expiresAt',
+  value:         'expiresAt',
+  labelKey:      'tableHeaders.expiresAt',
+  align:         'left',
+  width:         200,
+  formatter:     'Date',
+  formatterOpts: { emptyTextKey: 'generic.never' },
+};
+
+export const CREATED = {
+  value:         'metadata.creationTimestamp',
+  labelKey:      'tableHeaders.created',
+  align:         'left',
+  width:         150,
+  formatter:     'LiveDate',
+  formatterOpts: { addSuffix: true },
+};
```

Comment:
Vue UI uses "Age" (defined in here as `AGE`) instead of "Created"

Reply:
Okay, switched and removed Created column defn